qiskit_experiments/test/iq_pulse_backend.py

```diff
@@ -0,0 +1,195 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""A Pulse simulation backend based on Qiskit-Dynamics"""
+import datetime
+from typing import Union
+
+import numpy as np
+from qiskit import QuantumCircuit
+from qiskit.providers import BackendV2
+from qiskit.providers.models import PulseDefaults
+from qiskit.providers.options import Options
+from qiskit.qobj.utils import MeasLevel, MeasReturnType
+from qiskit.quantum_info.states import Statevector
+from qiskit.result import Result
+from qiskit.transpiler import Target
+
+from qiskit_dynamics import Solver
+from qiskit_dynamics.pulse import InstructionToSignals
+
+from qiskit_experiments.exceptions import QiskitError
+from qiskit_experiments.test.utils import FakeJob
+
+
+class IQPulseBackend(BackendV2):
+    """Pulse Simulator abstract class"""
+
+    def __init__(self, static_hamiltonian, hamiltonian_operators, dt=0.1 * 1e-9, **kwargs):
+        """Hamiltonian and operators is the Qiskit Dynamics object"""
+        super().__init__(
+            None,
+            name="PulseBackendV2",
+            description="A PulseBackend simulator",
+            online_date=datetime.datetime.utcnow(),
+            backend_version="0.0.1",
+        )
+        self._defaults = PulseDefaults.from_dict({})
```

Comment:
We can use this to have a default instruction schedule map.
```suggestion
        self._pulse_defaults = PulseDefaults.from_dict(self.__pulse_config__)
```

Reply:
`self._defaults` is requried to run the experiments. As for the `_pulse_defaults` shouldn't this be done using `cmd_def`